CHANGES

```diff
@@ -23,15 +26,21 @@ the Git ChangeLog, the GitHub repo https://github.com/ProofGeneral/PG
      modules as coqdep error.
 *** Renew support for proof-tree visualization, see description in
     generic changes above.
-
-
-**** New options coq-compile-extra-coqc-arguments and
-     coq-compile-extra-coqdep-arguments to configure additional
-     command line arguments to calls of, respetively, coqc and coqdep
-     during auto compilation.
-
-**** Fix issues #687 and #688 where the omit-proofs feature causes
-     errors on correct code.
+*** New command `proof-check-proofs' to generates the proof status
+    of all opaque proofs. This command is useful for a development
+    process where invalid proofs are permitted and vos compilation and
+    the omit proofs feature are used to work at the most interesting
+    or challenging point instead of on the first invalid proof. The
+    command generates a list of all opaque proofs in the current
+    buffer together with the information whether the proof script is
+    currently valid or invalid. The command can also be run in batch
+    mode, for instance in a continuous integration environment.
```

Comment:
CI is a good use case as well indeed. Did you already tried this? I mean, not from an internal PG elisp test, but from an example GHA script that would call emacs like this? `
```
emacs --batch --eval "(progn (setq user-init-file \"~/.emacs\"))" --load ~/.emacs --eval "(message \"etc.\")"
```

And maybe I'm mistaken, but to facilitate this, maybe the function `proof-check-proofs` should query from some way the **filename** of the file to test? (which could be the current buffer in interactive mode, for example)

Reply:
No GHA and no other repo but I manually tested it a lot from the shell. Please see the PG manual for the command line. Adding a test invoking `proof-check-proofs` from a makefile is probably a good idea.

Reply:
I investigated the error yesterday: 3-pane mode (on by default) is broken if the frame height is too small and certain Emacses set the frame height to 9 inside a docker container.
In the commit I added yesterday, the test you asked for runs as part of the simple tests. Do you have the opinion that this is not enough?

Reply:
> certain Emacses set the frame height to 9 inside a docker container.

OK!

> In the commit I added yesterday, the test you asked for runs as part of the simple tests. Do you have the opinion that this is not enough?

No worries! it is enough as is.

Thanks a lot @hendriktews !